code/game/gamemodes/wizard/artefact.dm

```diff
@@ -196,8 +196,12 @@
 
 /obj/item/phylactery/examine(mob/user, size, show_name)
 	..()
-	if(iswizard(user))
-		to_chat(user, "<span class='sinister'>You can use charged soulstones to refill it. The more charges you have, the faster you will revive.</span>")
+	if(user.mind == bound_mind)
+		to_chat(user, "<span class='notice'>It has [charges ? "[charges]" : "no"] charges left!</span>")
+	if(iswizard(user)) //Wizards are scholars
+		to_chat(user, "<span class='notice'>This is a phylactery! Whoever is bound to it will come back as a lich wherever the phylactery may be, as long as it is charged.</span>")
+		to_chat(user, "<span class='notice'>You can charge it using filled soulstones. The more charges you have, the faster you will revive.</span>")
+		to_chat(user, "<span class='notice'>The phylactery will revive you about ten times faster if you are close to it when perishing.</span>")
```

Comment:
This phrasing is weird; it's not "About" ten times faster, it's exactly ten times faster. Why the ambiguity over closeness? Close to me sounds like maybe the phylactery is visible on screen, so distance 7, or maybe it needs to be adjacent, so distance 1.

Frankly I think it would be harmless to move it up to 7 distance, or even scale the revive time entirely based on distance (e.g.: revive time is divided by `clamp(1,16-distance*2,10)`, so 10x at distance 1-3, 8x at 4, 6x at 5, 4x at 4, 2x at 6, and 1x at 7. You don't have to do this to get a merge, but I would like you to make the message less ambiguous if you are keeping it as-is.

Reply:
Sure